lxd/instance/drivers/driver_qemu.go

```diff
@@ -96,7 +96,7 @@ const qemuPCIDeviceIDStart = 4
 // qemuDeviceIDPrefix used as part of the name given QEMU devices generated from user added devices.
 const qemuDeviceIDPrefix = "dev-lxd_"
 
-// qemuDeviceNamePrefix used as part of the name given QEMU blockdevs and netdevs generated from user added devices.
+// qemuDeviceNamePrefix used as part of the name given QEMU blockdevs, netdevs and device tags generated from user added devices.
```

Comment:
This commit modifies a line already changed in the earlier commit.
Instead this change should be squashed into the earlier commit so that the commits tell a logical story of the changes (in a meaningful order) rather than an evolution of your work.

Reply:
Sorry, I am confused in how I should approach this. In the first commit this variable was repurposed to include netdevs names and after the second it started being used for device tags as well, this is why the comment is changed twice. I could squash both changes on one commit if you think this would be best.

Reply:
Oh right I see you've made 2 discrete changes to it. OK leave as is. I had expected a single change for this PR, but it sounds like you're fixing several things at once in this PR via separate commits.

Reply:
Thanks makes sense, please include this sort of thing in commit messages as it helps the reviewer understand the reason behind the changes. Ta